packages/grafana-ui/src/components/TagsInput/TagItem.tsx

```diff
@@ -12,18 +12,33 @@ interface Props {
   name: string;
   disabled?: boolean;
   onRemove: (tag: string) => void;
+  useThemeColors?: boolean;
 }
 
 /**
  * @internal
  * Only used internally by TagsInput
  * */
-export const TagItem = ({ name, disabled, onRemove }: Props) => {
-  const { color, borderColor } = useMemo(() => getTagColorsFromName(name), [name]);
+export const TagItem = ({ name, disabled, onRemove, useThemeColors }: Props) => {
```

Comment:
I think this is fine, but I'm not a fan of this name. 

I associate this change with _removing_ colored tags, not _using_ other colours.

maybe we could do like `<TagItem randomColors={false} />` or `<TagItem colors="random" />`/`<TagItem colors="themed"/>` ?

Reply:
I see what you mean. If this change is about "removing colored tags" or "disabling colored tags" and the default value is random colors what do you think about `disableColoredTags` ? This way it make more explicit the behavior of the boolean.